config/cmake/config.hpp.in

```diff
@@ -98,6 +98,9 @@
 // Enable MFEM functionality based on Conduit
 #cmakedefine MFEM_USE_CONDUIT
 
+// Enable MFEM functionality based on the SCOREC library
```

Comment:
Is it `SCOREC` library or `SCOREC PUMI` library, or just `PUMI`?

Reply:
Corrected. We use `PUMI` library.